cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/sidebar_filtering.spec.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import {
+  DATASOURCE_NAME,
+  INDEX_PATTERN_NAME,
+  INDEX_NAME,
+  START_TIME,
+  END_TIME,
+  WORKSPACE_NAME,
+} from './constants.js';
+import * as dataExplorer from './helpers.js';
+import { SECONDARY_ENGINE, BASE_PATH } from '../../../../../utils/constants';
+
+const miscUtils = new MiscUtils(cy);
```

Comment:
I don't remember the exact reasoning, but we haven't been using this. Instead we were using cy custom commands directly. I'll comment below where relevant